src/index.js

```diff
@@ -2,6 +2,16 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
+import './fonts/gilroy/Gilroy-Black.ttf';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/The-Major-Project/Design-and-Develop-frontend/pullRequest?prid=8041234)
```suggestion
import "./fonts/gilroy/Gilroy-Black.ttf";
```